src/error.ts

```diff
@@ -1018,6 +1018,34 @@ export class MongoTopologyClosedError extends MongoAPIError {
   }
 }
 
+/**
+ * An error generated when the MongoClient is closed and async
+ * operations are interrupted.
+ *
+ * @public
+ * @category Error
+ */
+export class MongoClientClosedError extends MongoAPIError {
+  /**
+   * **Do not use this constructor!**
+   *
+   * Meant for internal use only.
+   *
+   * @remarks
+   * This class is only meant to be constructed within the driver. This constructor is
+   * not subject to semantic versioning compatibility guarantees and may change at any time.
+   *
+   * @public
+   **/
+  constructor(message = 'Operation interrupted because client was closed') {
```

Comment:
```suggestion
  constructor() {
```

suggest: don't bother with a message, we only construct this in one place with the same error every time.

Reply:
No message at all? that seems unconventional

Reply:
Oh gotcha